### ISSUE TITLE
Update error V1.4.0-19: "Port undefined configured for this package"

### ISSUE THREAD

**Issue by endbacher** [kastelo/syncthing-synology#3]
I have Syncthing v1.3.3.3 installed & running on my Diskstation 415+ and packet centre prompts for an update to 1.4.0-19. File is properly downloaded but gives the following message on install:
“Port undefined configured for this package is either used by another service or reserved for system use. Please disable or modify the conflicting service, or contact the developer to modify the package configuration”
I already tried the following:
- disabling the Syncthing installation prior to update 
- rebooted without Syncthing running
- changed port in the GUI settings
but wasn't successful.

Does anybody know what might be the reason and how this could be solved?
Thanks in advance.

**Comment by calmh** [comment]
1.4.0-19 is not a package from here. I have no idea what the problem might be.

**Comment by endbacher** [comment]
Thanks for the hint, I did not realize that the offered 1.4.0-19 update is the one from SynoCommunity being also an active package source on the device.

@calmh 
Is there any way to separate the two independent syncthing versions of kastelo and SynoCommunity in a better way to avoid interfering with each other? 

Two issues I experienced:

- with package from SynoCommunity initially installed ([https://synocommunity.com/package/syncthing](https://synocommunity.com/package/syncthing)) and adding the kastelo package source afterwards it upgraded the existing syncthing package on device killing the old config so a full new setup of the sync destinations was required for the kastelo version.

- now with kastelo package installed it offers update as per above so perhaps I was just lucky it did not install with regards of the different configuration setup of the two packages?

Thanks.
